vortexor/src/vortexor.rs

```diff
@@ -0,0 +1,216 @@
+use {
+    crossbeam_channel::Sender,
+    solana_net_utils::{bind_in_range_with_config, bind_more_with_config, SocketConfig},
+    solana_perf::packet::PacketBatch,
+    solana_sdk::{quic::NotifyKeyUpdate, signature::Keypair},
+    solana_streamer::{
+        nonblocking::quic::DEFAULT_WAIT_FOR_CHUNK_TIMEOUT,
+        quic::{spawn_server_multi, EndpointKeyUpdater, QuicServerParams},
+        streamer::StakedNodes,
+    },
+    std::{
+        net::UdpSocket,
+        sync::{atomic::AtomicBool, Arc, Mutex, RwLock},
+        thread::{self, JoinHandle},
+        time::Duration,
+    },
+};
+
+pub const MAX_QUIC_CONNECTIONS_PER_PEER: usize = 8;
```

Comment:
Should we move these two const to cli.rs and rename to add `DEFAULT`? Currently it's giving an impression that these are the hard limits.

Reply:
done